web/js/app.js

```diff
@@ -376,25 +389,285 @@ LR.utils.preSelectMultiSelects = function(annotationData, keys, form){
         if(annotationData.hasOwnProperty(key)){
             let data_arr = annotationData[key].hasOwnProperty("value") ? annotationData[key].value.items : annotationData[key].items
             let input = form.querySelector("input[deer-key='"+key+"']")
-            let sel = input.nextElementSibling //The view or select should always be just after the input tracking the values from it.
-            if(sel.tagName !== "SELECT"){
+            let area = input.nextElementSibling //The view or select should always be just after the input tracking the values from it.
```

Comment:
I'm trying to follow exactly what cases you are covering here. The use of `Node.closest()` might be advisable.

Reply:
Yes, this is depending on the order of things in the HTML templates.  Certainly not a good way to do it, `closest()` would be an improvement.